docs/README.md

```diff
@@ -72,7 +69,6 @@ Here follows the list of GitHub Actions topics available in the current document
     - [setup-github-release-binary](#setup-github-release-binary)
     - [setup-java-build](#setup-java-build)
     - [setup-kind](#setup-kind)
-    - [travis-env-load](#travis-env-load)
```

Comment:
if you install markdown all in one vscode it automatically update this TOC, otherwise make sure it's linked as before